deployment/ansible/raster-foundry-jenkins.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: jenkins
+  connection: ssh
+  become: True
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes cache_valid_time=3600
+      become: true
+
+  roles:
+    - role: raster-foundry.jenkins
```

Comment:
Consistency nitpick here with missing curly braces around `role`, but overall I like the approach of wrapping the `azavea.jenkins` role.